packages/strapi-plugin-upload/config/schema.graphql.js

```diff
@@ -57,6 +56,15 @@ module.exports = {
           return await strapi.plugins.upload.services.upload.updateFileInfo(id, info);
         },
       },
+      deleteFile: {
+        description: 'Delete one file',
+        resolverOf: 'plugins::upload.upload.destroy',
+        resolver: async (obj, options, { context }) => {
+          const file = await strapi.plugins.upload.services.upload.fetch({ id: context.params.id });
```

Comment:
You will just need to check the file exist before trying to remove it

Reply:
I think you need to add a test on the file and throw a notFound error

Reply:
I added a test, but for some reason there is no error. I'll investigate.

Reply:
I don't know if the test is relevant, but I fixed it at least!